builds/azure-pipelines/template-steps-build-test.yml

```diff
@@ -174,13 +174,19 @@ steps:
     GdnPublishTsaConfigFile: '$(Build.SourcesDirectory)\builds\TSAConfig.gdntsa'
   condition: and(succeeded(), eq(variables['Agent.OS'], 'Windows_NT'), eq(variables['TSA_UPLOAD'], 'true'))
 
-# 5.0 isn't supported on Mac yet
+# net6+ isn't supported on Mac yet
```

Comment:
Is .net core 2.1 really still needed for mac?

Reply:
I'll check once I've gotten the other issues fixed 😄